src/cascadia/TerminalApp/TabRowControl.xaml

```diff
@@ -11,96 +11,117 @@
-                                                ResourceKey="TabViewButtonBackground" />
-                                <StaticResource x:Key="SplitButtonForeground"
-                                                ResourceKey="TabViewButtonForeground" />
-                                <StaticResource x:Key="SplitButtonBackgroundPressed"
-                                                ResourceKey="TabViewItemHeaderBackgroundPressed" />
-                                <StaticResource x:Key="SplitButtonForegroundPressed"
-                                                ResourceKey="TabViewItemHeaderForegroundPressed" />
-                                <StaticResource x:Key="SplitButtonBackgroundPointerOver"
-                                                ResourceKey="TabViewItemHeaderBackgroundPointerOver" />
-                                <StaticResource x:Key="SplitButtonForegroundPointerOver"
-                                                ResourceKey="TabViewItemHeaderForegroundPointerOver" />
-                            </ResourceDictionary>
-                            <ResourceDictionary x:Key="HighContrast">
-                                <StaticResource x:Key="SplitButtonBackground"
-                                                ResourceKey="TabViewButtonBackground" />
-                                <StaticResource x:Key="SplitButtonForeground"
-                                                ResourceKey="TabViewButtonForeground" />
-                                <StaticResource x:Key="SplitButtonBackgroundPressed"
-                                                ResourceKey="TabViewItemHeaderBackgroundPressed" />
-                                <StaticResource x:Key="SplitButtonForegroundPressed"
-                                                ResourceKey="TabViewItemHeaderForegroundPressed" />
-                                <StaticResource x:Key="SplitButtonBackgroundPointerOver"
-                                                ResourceKey="TabViewItemHeaderBackgroundPointerOver" />
-                                <StaticResource x:Key="SplitButtonForegroundPointerOver"
-                                                ResourceKey="TabViewItemHeaderForegroundPointerOver" />
-                            </ResourceDictionary>
-                        </ResourceDictionary.ThemeDictionaries>
-                    </ResourceDictionary>
-                </mux:SplitButton.Resources>
-            </mux:SplitButton>
-        </mux:TabView.TabStripFooter>
+        <ContentPresenter x:Name="ElevatedShield"
+                          Grid.Column="0">
 
-    </mux:TabView>
+            <!--  EA18 is the "Shield" glyph  -->
+            <FontIcon x:Uid="ElevationShield"
+                      Margin="9,4,0,0"
+                      FontFamily="Segoe MDL2 Assets"
+                      FontSize="16"
+                      Glyph="&#xEA18;"
+                      Visibility="{x:Bind ShowUacShield, Mode=OneWay}" />

```

Comment:
You can remove the `OneWay` property. There's no way for this value to change during runtime, right?

Reply:
Well, someone could disable the shield via the setting. This lets it hot-reload